app/src/main/java/com/github/sdpteam15/polyevents/helper/GoogleMapHelper.kt

```diff
@@ -426,4 +432,42 @@ object GoogleMapHelper {
         }
         tempPoly?.points = tempLatLng
     }
+
+
+    fun areasToFormattedStringLocations(
+        points: MutableMap<Int, List<LatLng>> = coordinates,
+        from: Int = 0,
+        to: Int = points.size
+    ): String {
+        println("Range " + (from until to).toString())
+        println("From $from")
+        println("To $to")
+        println("Map $points")
+
+        var s = ""
+        for (i in from until to) {
+            s += areaToFormattedStringLocation(points[i])
+            s += DatabaseConstant.AREAS_SEP
+        }
+        println(s)
+        return s.substring(0, s.length - DatabaseConstant.AREAS_SEP.length)
+    }
+
+    fun areaToFormattedStringLocation(loc: List<LatLng>?): String {
+        if (loc == null){
+            return ""
+        }
+        var s = ""
+
+        for (c in loc) {
+            s += c.latitude.toString() + DatabaseConstant.LAT_LONG_SEP + c.longitude + DatabaseConstant.POINTS_SEP
```

Comment:
Since c.latitude has a .toString() after it, should c.longitude have it too ?

Reply:
Yep, you're right !